src/utilities/utilities.js

```diff
@@ -113,4 +114,17 @@ module.exports = class Utilities {
   static firstCharToLowercase (str) {
     return str[0].toLowerCase() + str.substr(1)
   }
+
+  static difference (a, b) {
+    return [
+      a.filter((item) => !b.includes(item)),
+      b.filter((item) => !a.includes(item))
+    ]
+  }
+
```

Comment:
Although these functions are being indirectly invoked by tests elsewhere, should they have their own unit tests so that they are being properly verified?